src/components/Match/BuildingMap/buildingData.js

```diff
@@ -1,11 +1,11 @@
 export default [
   {
     id: 't4br',
-    style: 'position: absolute; top: 81%; left: 12%;',
+    style: 'position: absolute; top: 82%; left: 12%;',
```

Comment:
if you are messing with all these values anyway, you might want to convert it from CSS to an object since parsing this is kind of messy.  I skipped that for dev speed in my initial implementation.